pages/osx/date.md

```diff
@@ -0,0 +1,19 @@
+# date
+
+> Set or display the system date.
+
+- Display the date using the default locale:
+
+`date +"%c"`
+
+- Display the date in UTC and ISO 8601 format:
+
+`date -u +"%Y-%m-%dT%H:%M:%SZ"`
+
+- Display the number of seconds since the unix epoch:
+
+`date +%s`
+
+- Convert a unix expoch timestamp to a date:
```

Comment:
We can be creative and use words that have (ideally start with) those letters, even if that wasn't the origin of the abbreviation, as long as the meaning is made clearer for tldr users :)